org.lflang.lfc/test/resources/org/lflang/lfc/tests/tabs.stderr

```diff
@@ -0,0 +1,16 @@
+lfc: error: State must have a type.
+--> test/resources/org/lflang/lfc/tests/tabs.lf:4:2
+  |
+3 | main reactor {
+4 |     state foo(1);
+  |     ^^^^^^^^^^^^ State must have a type.
+  |
+5 |     state    bar(1);
+lfc: error: State must have a type.
+--> test/resources/org/lflang/lfc/tests/tabs.lf:5:2
+  |
+4 |     state foo(1);
+5 |     state    bar(1);
+  |     ^^^^^^^^^^^^ State must have a type.
```

Comment:
actually there's a problem here which I should fix. The first warning also spans over the `(1)`, which I think is the correct behavior. This one doesn't.